tests/net/socket/tcp/testcase.yaml

```diff
@@ -7,6 +7,8 @@ common:
     - userspace
   filter: CONFIG_FULL_LIBC_SUPPORTED
   timeout: 180
+  platform_exclude:
+    - nrf5340dk/nrf5340/cpuapp/ns
```

Comment:
could `min_flash` be used? Since there is a `min_ram` already

Reply:
Perhaps, I can check.

Reply:
It is the tfm configuration that affects the flash usage. So `nrf5340dk/nrf5340/cpuapp` works just fine, but the non secure version does not. Not sure how the `min_flash` could be set in this case.

Reply:
In `boards/nordic/nrf5340dk/nrf5340dk_nrf5340_cpuapp.yaml` is:
```
ram: 448
flash: 1024
```
in `boards/nordic/nrf5340dk/nrf5340dk_nrf5340_cpuapp_ns.yaml` is:
```
ram: 192
flash: 192
```

Reply:
Ok, lets go that route then. I submitted new version using min_flash